acceptance/tests/parser_functions/hiera_in_templates.rb

```diff
@@ -0,0 +1,403 @@
+test_name "Calling Hiera function from inside templates"
+
+# Global variables
+
+$module_name = "hieratest"
```

Comment:
these don't have to be globals, nor should they be.  variables inside this test's namespace are fine. (remove the '$')